### PR TITLE
DG-KK Styling 20160507

### PR DESCRIPTION
**Pull request by charlieartist**

### CODE REVIEW

File: docroot/sites/all/themes/custom/govcon_2016/sass/components/_sponsors.scss [hunk 1]
```diff
@@ -13,15 +14,20 @@
...
+  background-color: rgba(255,255,255,1);
+  border-top: 4px solid $gold;
   padding: 0 0 16px;
   margin-left: -1em;
   margin-right: -1em;
 
   .bg { background-color: $gold; }
   h2 { @include hide-text; }
 }
+
```
Comment (aweingarten) [inline docroot/sites/all/themes/custom/govcon_2016/sass/components/_sponsors.scss:28]: Extra lines unnecessary.

File: docroot/sites/all/themes/custom/govcon_2016/sass/components/_sponsors.scss [hunk 2]
```diff
@@ -13,15 +14,20 @@
    -webkit-font-smoothing: antialiased;
 }
 .pane-cod-sponsors-panel-pane-1 { 
-  margin-top: 50px;
-  background-color: $gold;
+  //margin-top: 50px;
+  //background-color: $gold;
+  background-color: rgba(255,255,255,1);
```
Comment (aweingarten) [inline docroot/sites/all/themes/custom/govcon_2016/sass/components/_sponsors.scss:19]: In theory should be a variable.

File: docroot/sites/all/themes/custom/govcon_2016/sass/components/_sponsors.scss [hunk 3]
```diff
@@ -13,15 +14,20 @@
    -webkit-font-smoothing: antialiased;
 }
 .pane-cod-sponsors-panel-pane-1 { 
-  margin-top: 50px;
-  background-color: $gold;
+  //margin-top: 50px;
```
Comment (aweingarten) [inline docroot/sites/all/themes/custom/govcon_2016/sass/components/_sponsors.scss:17]: We don't to commit commented out code ideally.

File: docroot/sites/all/themes/custom/govcon_2016/sass/components/_sponsors.scss [hunk 4]
```diff
@@ -91,7 +97,7 @@
 
 @include breakpoint($wide, true) {
   .node-type-event .pane-node-body {
-    @include span(10 of 12);
+    //@include span(10 of 12);
```
Comment (aweingarten) [inline docroot/sites/all/themes/custom/govcon_2016/sass/components/_sponsors.scss:100]: Same here.  Don't want commented code committed.

File: docroot/sites/all/themes/custom/govcon_2016/sass/layouts/govcon-simple/govcon-simple.layout.scss [hunk 5]
```diff
@@ -46,6 +50,14 @@
   font-size: 1.2em;
 }
 
+body {
```
Comment (aweingarten) [inline docroot/sites/all/themes/custom/govcon_2016/sass/layouts/govcon-simple/govcon-simple.layout.scss]: Remove commented out code and unused tags.

File: docroot/sites/all/themes/custom/govcon_2016/sass/components/_footer.scss [hunk 6]
```diff
@@ -1,10 +1,11 @@
 
 .footerwrapper {
   background: rgba(68,91,73,1);
+  padding-bottom: 1em;
 }
 
 .l-footer {
-  margin-bottom: 0;
+  margin-bottom: 0; // This needs overriding in the theme layout!
```
Comment (aweingarten) [inline docroot/sites/all/themes/custom/govcon_2016/sass/components/_footer.scss]: Lets remove this

**Comment by charlieartist** [comment]
Steps to complete:
- Replace curent Leaflet library with attached library.
- Diamond Sponsors: 
  https://capitalcampdev.prod.acquia-sites.com/node/1/panelizerSponsors
  grouped by sponsorship level (diamond). Add these CSS classes to the display: `diamond_banner_group`
- Homepage Banner and Location Map: Panels visibility rule <front> doesn't work on COD. To display, it needs to be changed to: String: URL path: drupal-govcon-2016
- Current sponsors view disabled
- Some modifications to Panelizer--but this may need to be changed, since it broke some items.

[leaflet.zip](https://github.com/Drupal4Gov/Drupal-GovCon-2016/files/257853/leaflet.zip)

**Comment by charlieartist** [comment]
[x] [Visit the URL Here ](https://capitalcampdev.prod.acquia-sites.com/admin/structure/pages/nojs/operation/site_template/handlers/site_template__panel_context_1cbf1e90-2219-4dbc-9dd1-8de9961d3591/content) 

[x] Click on the gear by the top right corner of "Homepage Banner"
[x] Under visibility rules click "Frontpage"
[x] Click remove.
[x] Click on the gear by the top right corner of "Homepage Banner"
[x] Under visibility rules click "add new rule"
[x] select "string url path" and enter `drupal-govcon-2016`

[x] Click on the gear by the top right corner of "location map"
[x] Under visibility rules click "Frontpage"
[x] Click remove.
[x] Click on the gear by the top right corner of "Homepage Banner"
[x] Under visibility rules click "add new rule"
[x] select "string url path" and enter `drupal-govcon-2016`

[x] Click on the gear by the top left corner of "First Lower Content Area"
[x] Click add content
[x] Click view pane
[x] Select "Sponsors grouped by sponsorship level"
[x] Select "diamond" in the "Sponsorship Level"
[x] Click finish
[x] Click on the gear by the top right corner of "Sponsors grouped by sponsorship level" 
[x] select css classes
[x] under css classes field enter `diamond_banner group`
[x] click save

[x] go to `https://capitalcampdev.prod.acquia-sites.com/node/1/panelizer/default/content`
[x] click the gear on "Sponsors grouped by sponsorship level" 
[x] disable the pane
[x] click save
[x] click the gear on "Custom: Drupal GovCon Twitter feed" 
[x] disable the pane
[x] click save
[x] click the gear on "Sponsors not grouped" 
[x] disable the pane
[x] click save
[x] move "view News Multi News Item pane" into the preface region
[x] click save

[] go to https://capitalcampdev.prod.acquia-sites.com/node/add/banner-image
[] enter the title of "Welcome To GovCon 2016"
[] enter the body of
`<h2>Announcing Drupal GovCon 2016</h2>
<p>
<a href="#">Register</a>
<a href="#">Submit Session</a>
</p>`
[] select display location of "home"
[] Click save.
- Replace curent Leaflet library with attached library (done in sites/all/libraries/leaflet).
- Diamond Sponsors: https://capitalcampdev.prod.acquia-sites.com/node/1/panelizerSponsors grouped by sponsorship level (diamond). Add these CSS classes to the display: diamond_banner_group
- Homepage Banner and Location Map: Panels visibility rule doesn't work on COD. To display, it needs to be changed to: String: URL path: drupal-govcon-2016. This is done at http://local.drupalgovcon.org/admin/structure/pages/nojs/operation/site_template/handlers/site_template__panel_context_1cbf1e90-2219-4dbc-9dd1-8de9961d3591/content for both the View: Banner Image: Homepage Banner in the Banner area, and View: Location Map: Attachment in the Second Lower Content.
- Current sponsors view disabled
- Some modifications to Panelizer--but this may need to be changed, since it broke some items. I would need to see a rollback in order to find out what happened.

**Pull request merged**